api/src/certification/session-management/domain/read-models/InvigilatorSession.js

```diff
@@ -0,0 +1,16 @@
+class InvigilatorSession {
+  constructor({ finalizedAt, invigilatorPassword }) {
+    this.invigilatorPassword = invigilatorPassword;
+    this.finalizedAt = finalizedAt;
+  }
+
+  isAccessible() {
+    return this.finalizedAt === null;
```

Comment:
Si par hasard finalizedAt etait passe a undefined, le `===` renverrait l'inverse
```suggestion
    return !!this.finalizedAt;
```